tst/PeerConnectionFunctionalityTest.cpp

```diff
@@ -134,6 +134,20 @@ TEST_F(PeerConnectionFunctionalityTest, exchangeMedia)
         return;
     }
 
+    // Create track and transceiver and adds to PeerConnection
```

Comment:
I think you can remove? Maybe the diff tool is broken, but I think this always was here!

Reply:
Yeah! Has always been here. Not sure what happened.